.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear description of what the bug is, error log.
+
+**To Reproduce**
+A reproducer should list:
+1. compilers (with versions)
+2. Kokkos release or commit used
+3. platform/backends (e.g. HIP-AMD, or CUDA, OpenMP, etc)
+4. minimal code that reproduces the bug
+
```

Comment:
```suggestion
5. command line used to reproduce the bug
```

5 may be covered by 4. above but I think adding 4 makes this clearer.

Reply:
@e10harvey -- Good point.  I'll incorporate your changes.  Many thanks for the review!